charts/apbs-rest/templates/id-ingress.yaml

```diff
@@ -31,9 +31,9 @@ spec:
     - host: {{ . }}
       http:
         paths:
-          - path: /id(/?)$
+          # - path: /id(/?)$
           # TODO: 2020/07/02, Elvis - commenting out line below for now since user doesn't need access beyond retrieving a new ID
-          # - path: /id(/|$)(.*)
```

Comment:
Similar feelings here as with the rewrite targets since the validate path is only needed for services within the cluster